src/window/window_wrapper/mod.rs

```diff
@@ -222,7 +222,10 @@ pub fn start_loop(
         .with_decorations(!SETTINGS.get::<CmdLineSettings>().frameless);
 
     #[cfg(target_os = "linux")]
-    let winit_window_builder = winit_window_builder.with_app_id("Neovide".to_string());
+    let winit_window_builder = winit_window_builder
+        .with_app_id(SETTINGS.get::<CmdLineSettings>().wayland_app_id)
+        .with_class("neovide".to_string(), SETTINGS.get::<CmdLineSettings>().x11_wm_class
```

Comment:
https://docs.rs/winit/0.20.0-alpha4/winit/platform/unix/trait.WindowBuilderExtUnix.html#tymethod.with_class

you made "class" static, and now changing "instance" instead.

Reply:
https://docs.rs/winit/0.25.0/winit/window/struct.WindowBuilder.html#method.with_class
with the newer version of winit this seems to have switched places

Reply:
Oh, i see, great :)

Reply:
It had me too :D
I just double checked, Kethku/winit `new-keyboard-all` branch reports it's based on version 0.24.0, and this already has the same structure as 0.25.0 https://docs.rs/winit/0.24.0/winit/window/struct.WindowBuilder.html#method.with_class :+1: